src/character/gear/EditableItemEntry.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useContext, useEffect, useState } from "react";
+import styled, { css } from "styled-components";
+import { Theme } from "../../Theme";
+import { CustomItem, CustomItemData } from "../../utils/types/CustomItem";
+import { Modal, TextArea } from "react-aria-components";
+import { CharacterContext } from "../../DeadlandsCompanion";
+import { Button } from "../../shared/buttons/Button";
+import { useLongPress } from "use-long-press";
+import { deleteCustomitem, updateCustomItem } from "./CustomItemService";
+
+const EditableItemEntryWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border-radius: 4px;
+  ${({}) => css`
+    background-color: ${Theme.Surface[300]};
+  `}
+`;
+type EditableItemEntryProps = {
+  item: CustomItem;
+};
+
+const ModalContents = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  align-items: center;
+  justify-content: center;
+`;
+
+const ItemTitle = styled.h4`
+  margin: 0;
+  padding: 0;
+  font-size: 1em;
+`;
+
+const ItemWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  padding: 4px;
+  border-radius: 8px;
+  justify-content: space-between;
+`;
+
+const ButtonsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  padding: 4px;
+  border-radius: 8px;
+  justify-content: flex-end;
+  button {
+    padding: 4px;
+    border-radius: 4px;
+    border: none;
+  }
```

Comment:
Any reason to overwrite the style of the buttons here? Note that the base component is intentionally oversized to make touch interaction easier.